File: IR LLVM/Micro04/Micro04.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int x;
    int num;
    int intervalo = 0;

    for(x = 1; x <= 5; x++){
        printf("Digite um numero: ");
        scanf("%d", &num);
        if(num >= 10){
            if(num <= 150)
                intervalo = intervalo + 1;
        }
    }
    printf("Ao total foram digitados %d numeros no intervalo 10 e 150\n", intervalo);
    return 0;
}

File: GeradorCodigo/codigo/Micro04.pas
program micro04;

var 

x, num, intervalo: integer;

begin
    for x:= 1 to 5 do
        begin
            write('Digite um numero: ');
            read(num);
            if (num >= 10) then
                begin
                    if (num <= 150) then
                        begin
                        intervalo := intervalo + 1;
                        end
                end
        end;
    write('Ao total, foram digitados, ');
    write(intervalo);
    writeln(', numeros no intervalo entre 10 e 150');
end.
